### ISSUE TITLE
README example actually has a bug! Oops!

### ISSUE THREAD

**Issue by thundergolfer** [thundergolfer/Insults#1]
Here is the readme example: 

```
>>> comments = ["You called me a 'dickhead', so I'll say you're a cunt.", "These shitakes taste like shit."]
>>> Insults.foul_language(comments, context=False)
```

It's supposed to return `['cunt', 'shit']` but it actually returns just `['shit']`. It might be the period isn't getting separated from the `cunt` token?

**Comment by thundergolfer** [comment]
This was noted in the source for the `foul_language` method. Basically the `nltk` method for sentence tokenisation kinda sucks. It takes something like `"dickhead"` and makes `["``", "dickhead", "''"]`. Why does it need to change the quote characters?! 

I think I will need to move to `spacy`'s for sentence tokenisation.

**Comment by thundergolfer** [comment]
Addressed in c0a160ad6bfb5335015ff0498d4535569a4e4554. Will close and open a new issue specifically about moving off `nltk` and getting on `spacy`.
